site/source/user-manual/service-guides/lightning/lightning-integrations.rst

```diff
@@ -151,19 +151,36 @@ Available For
 - Android
 - iOS
 
-.. note:: Compatible with LND only
+.. note:: Works with both LND and Core Lightning
```

Comment:
```suggestion
.. note:: Works with both LND and CLN
```

Reply:
Pretty sure we're calling this Core LIghtning now?

Reply:
So we're going to abbreviate LND, but not CLN?  Why?

Reply:
LND is actually called LND I believe, CLN is an abbreviation.